README.md

```diff
@@ -1,10 +1,12 @@
 # meta-cyclonedx
 
-`meta-cyclonedx` is a [Yocto](https://www.yoctoproject.org/) meta-layer which produces a [CycloneDX](https://cyclonedx.org/) Software Bill of Materials (aka [SBOM](https://www.ntia.gov/SBOM)) from your root filesystem.  
+`meta-cyclonedx` is a [Yocto](https://www.yoctoproject.org/) meta-layer which produces [CycloneDX](https://cyclonedx.org/) Software Bill of Materials (aka [SBOM](https://www.ntia.gov/SBOM)) from your root filesystem.
+
 This repository is forked from [BG Networks repository](https://github.com/bgnetworks/meta-dependencytrack) but differs by the following:
-- Removed support for DependencyTrack.
+- Removed direct integration with DependencyTrack.
```

Comment:
Originally we planned to remove dependency track integration because we don't use it. But maybe we can just put it back and make it optional based on presence or not of some variable.
What do you think of it ?

Reply:
I am perfectly happy with removing the direct integration with DependencyTrack. Having the direct integration just means you have a direct dependency to another service being available and running. Additionally, then you might have to take DT versioning into account for the availability of API endpoints.

So I am perfectly happy with separating the generation of the CycloneDX files from adding them into DT. As long as the CycloneDX files are valid and can be imported into DT later on I am happy

Reply:
Out of curiosity, may I ask what you are doing with the CycloneDX files instead? Do you use a different tool or do you just store them as is?

Or: Is there a reason why you do not use DependencyTrack?

Reply:
Ok so perfect :+1: 
We don't use Dependecy Track because we are using an internal scanning tool :)